infrastructure/main.tf

```diff
@@ -13,3 +13,60 @@ locals {
   prefix            = "${var.product}-ss"
   prefix_no_special = replace(local.prefix, "-", "")
 }
+
+# Resource Group
+
+resource "azurerm_resource_group" "rg" {
+  name     = "darts-api-dev"
+  location = "uksouth"
+  #tags     = var.resource_group_tags
+}
+
+# Application Insights
+
+resource "azurerm_application_insights" "appinsights" {
+  name                = "darts-api-appinsights-dev"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  application_type    = "web"
+  #tags                = var.common_tags
+
+  lifecycle {
+    ignore_changes = [
+      # Ignore changes to appinsights as otherwise upgrading to the Azure provider 2.x
+      # destroys and re-creates this appinsights instance
+      application_type,
+    ]
+  }
+}
+
+# this key vault is created in every environment, but preview, being short-lived,
+# will use the aat one instead
+# This section includes hardcoded variables that were in variables.tf ...
+# ... said variables were moved here due to a Jenkins objection
+module "key-vault" {
+  source              = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
+  product             = "darts"
+  env                 = ""
+  tenant_id           = ""
+  object_id           = ""
+  resource_group_name = azurerm_resource_group.rg.name
+  # https://github.com/hmcts/devops-azure-ad/blob/master/users/prod_users.yml
+  product_group_name          = "Key Vault product group name"
```

Comment:
You need to find out what should go in here. That's why you are getting 'Error: No group found matching specified filter (displayName eq 'Key Vault product group name' and securityEnabled eq true)' in your Jenkins build. Maybe ask Scott or if not Platops if they can help you here.